packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/advanced-features/sgx/manage/manage.service.js

```diff
@@ -0,0 +1,26 @@
+import indexOf from 'lodash/indexOf';
+import snakeCase from 'lodash/snakeCase';
+
+import { DISPLAYING_ORDER } from './manage.constants';
+
+export default class {
```

Comment:
**suggestion**: This service should be a plain object, there are only static methods in it.